README.md

```diff
@@ -69,57 +67,60 @@ Below are the methods this SDK exposes. See [Iterable's API Docs](https://api.it
 
-:rotating_light: Due to a limitation in Safari, web in-app messages displayed in Safari do not respect the value set for `handleLinks`. As a workaround, give your links a `target` (`target="_blank"` for new tab, `target="_top"` / `target="_parent"` for same tab).
+:rotating_light: Due to a limitation in Safari browsers, web in-app messages displayed in Safari will not be able to add an auto-generated close button and automatically fire `trackInAppClose` events with it for you.
 
 Close Button Options:
 
-| Property Name              | Description                                                                                | Value                      | Default       |
-| -------------------------- | ------------------------------------------------------------------------------------------ | -------------------------- | ------------- |
-| color                      | What color the button is (does not affect custom icons)                                    | `string`                   | `undefined`   |
-| iconPath                   | Custom pathname to an image or SVG you want to show instead of the default "X"             | `string`                   | `undefined`   |
-| position                   | Where to appear relative to the in-app message                                             | `'top-right \| 'top-left'` | `'top-right'` |
-| isRequiredToDismissMessage | If `true`, prevent user from dismissing in-app message by clicking outside of message      | `boolean`                  | `undefined`   |
-| sideOffset                 | How much space to leave between the button and side of the container                       | `string`                   | `'4%'`        |
-| size                       | How large to set the width, height, and font-size                                          | `string \| number`         | `24`          |
-| topOffset                  | How much space to leave between the button and the top of the container                    | `string`                   | `'4%'`        |
+| Property Name              | Description                                                                           | Value                      | Default       |
+| -------------------------- | ------------------------------------------------------------------------------------- | -------------------------- | ------------- |
+| color                      | What color the button is (does not affect custom icons)                               | `string`                   | `undefined`   |
+| iconPath                   | Custom pathname to an image or SVG you want to show instead of the default "X"        | `string`                   | `undefined`   |
```

Comment:
```suggestion
| iconPath                   | Custom pathname to an image or SVG to use (instead of the default "X")        | `string`                   | `undefined`   |
```